Automattic/wp-calypso
# UpsellNudge: Refactor Banner and UpsellNudge

#### Changes proposed in this Pull Request

* Tease apart `Banner` to take out functionality related to upsells
* Add functionality to `UpsellNudge` that was originally handled by `Banner`
* See #38778 

#### Testing instructions

Banners that are not upsells (some of these are inactive):
* client/my-sites/activity/activity-log/index.jsx
* client/blocks/jetpack-backup-creds-banner/index.jsx
* client/my-sites/activity/activity-log/rewind-unavailability-notice.jsx
* client/reader/following/main.jsx
* client/blocks/privacy-policy-banner/index.jsx
* client/me/pending-payments/index.jsx

Previous tickets with testing instructions for UpsellNudges across Calypso:
* https://github.com/Automattic/wp-calypso/pull/41892
* https://github.com/Automattic/wp-calypso/pull/41557
* https://github.com/Automattic/wp-calypso/pull/41552
* https://github.com/Automattic/wp-calypso/pull/41545
* https://github.com/Automattic/wp-calypso/pull/41453
* https://github.com/Automattic/wp-calypso/pull/41451
* https://github.com/Automattic/wp-calypso/pull/41419
* https://github.com/Automattic/wp-calypso/pull/41311
* https://github.com/Automattic/wp-calypso/pull/40988
* https://github.com/Automattic/wp-calypso/pull/40721
* https://github.com/Automattic/wp-calypso/pull/40626
